build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
3.6.0 brings some helpful additions to the maven publishing Gradle stuff (to be utilized in a future PR)